Sources/Sentry/Public/SentryReplay.h

```diff
@@ -0,0 +1,50 @@
+#import <Foundation/Foundation.h>
+
+#if __has_include(<Sentry/SentryDefines.h>)
+#    import <Sentry/SentryDefines.h>
+#else
+#    import <SentryWithoutUIKit/SentryDefines.h>
+#endif
+
+#if SENTRY_TARGET_REPLAY_SUPPORTED
+
+@class UIView;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryReplay : NSObject
+
+/**
+ * Marks this view to be redacted during replays.
+ *
+ * @warning This is an experimental feature and may still have bugs.
+ */
+- (void)replayRedactView:(UIView *)view;
```

Comment:
nit: I think we can drop the `replay` part now, and just have `redactView` since it's under the SentryReplay namespace already